pykechain/client.py

```diff
@@ -1105,6 +1105,106 @@ def create_activity(
             parent._cached_children.append(new_activity)
         return new_activity
 
+    def clone_activities(
+        self,
+        activities: List[Union[Activity2, Text]],
+        parent: Union[Activity2, Text],
+        activity_update_dicts: Optional[Dict] = None,
+        clone_parts: Optional[bool] = False,
```

Comment:
every time i look at this I think of renaming this to `clone_part_models`. What do you think? Is that wise. Impact is basically, backend signatures of some functions/docs, tests, pykechain and frontend implementation but all relatively small. (this is the time to do it.)

Reply:
My 2 cents on logical naming of the API:

- activity_parent_id (added "activity")
- clone_part_models
- clone_part_instances (added "part")
- clone_part_children
- excluded_part_ids (note the extra "d", plus I assume part instances can also be ignored?)
- part_parent_model_id
- part_parent_instance_id
- part_model_rename_template ("clone" is obvious)
- part_instance_rename_template (is there a reason why these 2 fields cant be merged?)
- activities

Reply:
Even better would be to use different terminology for booleans such as `clone_part_models`. I dont know of any conventions, but `clone` is both a verb and a noun, which makes the field ambiguous. I prefer a verb, such as `include_part_models` to clarify this is a boolean field.

Reply:
good points
will rename the following (while we can)

- [ ] activity_parent_id (added "activity")
- [ ] clone_part_models
- [ ] clone_part_instances (added "part")
- [ ] clone_part_children
- [ ] excluded_part_ids (note the extra "d", plus I assume part instances can also be ignored?)
- [ ] part_model_parent_id (flipped)
- [ ] part_instance_parent_id (flipped)
- [ ] part_model_rename_template ("clone" is obvious)
- [ ] part_instance_rename_template (is there a reason why these 2 fields cant be merged?)
- [ ] activities